src/pkg/caendr/caendr/models/datastore/wormbase.py

```diff
@@ -0,0 +1,145 @@
+import re
+
+from logzero import logger
+
+from caendr.models.error import InternalError
+
+
+
+class WormbaseVersion():
+
+    def __init__(self, value):
+
+        if value is None:
+            self._value = None
+
+        # Copy value of another Wormbase Version object
+        elif isinstance(value, WormbaseVersion):
+            self._value = value._value
```

Comment:
is value._value guaranteed to be == value.upper()

Reply:
I believe so, in normal usage: other than this line, `self._value` can only be set on lines 14 or 32, where it's set to `None` or `value.upper()` respectively.  This means it should be impossible to get a `WormbaseVersion` object with a lowercase string in `self._value`, unless the `_value` field is modified directly in an external source.  I can add a check for this just in case.

Reply:
Fixed in ac3c502ee2c29f8c394fed6d9977369d2c33405f